File: CS/CustomFunctionCriteriaOperator.Module/Module.cs
using System;
using System.Collections.Generic;

using DevExpress.ExpressApp;
using System.Reflection;


namespace CustomFunctionCriteriaOperator.Module {
   public sealed partial class CustomFunctionCriteriaOperatorModule : ModuleBase {
      public CustomFunctionCriteriaOperatorModule() {
         InitializeComponent();
         CurrentCompanyOidOperator.Register();
      }
   }
}

File: VB/CustomFunctionCriteriaOperator.Module/Module.vb
Imports System
Imports System.Collections.Generic

Imports DevExpress.ExpressApp
Imports System.Reflection


Namespace CustomFunctionCriteriaOperator.Module
   Public NotInheritable Partial Class CustomFunctionCriteriaOperatorModule
       Inherits ModuleBase

      Public Sub New()
         InitializeComponent()
         CurrentCompanyOidOperator.Register()
      End Sub
   End Class
End Namespace
